dds/DCPS/RTPS/Spdp.h

```diff
@@ -448,6 +448,8 @@ class OpenDDS_Rtps_Export Spdp
   DiscoveredParticipantMap participants_;
   RtpsDiscovery* disco_;
   DCPS::RcHandle<RtpsDiscoveryConfig> config_;
+  const double quick_resend_ratio_;
+  const DCPS::TimeDuration min_resend_delay_;
```

Comment:
Right. this could be a separate PR if needed, but I noticed high mutex contention for one of the RtpsDiscoveryConfigs during the aggressive Thrasher tests so I tried to reduce load by having Spdp copy / cache these values. Currently, nothing is updating these once they are configured, but every Spdp/Sedp transport associated with the config will repeatedly try to go grab these values.

Reply:
I don't think there is a reason to change these two variables.